roles/python/tasks/main.yml

```diff
@@ -48,27 +48,16 @@
   pip: name=Sphinx executable=pip3
   become: yes
 
-- name: create aliases for executing tests on regular database backends
+- name: create aliases for executing tests on regular and GIS database backends
   lineinfile:
     dest: "{{ user_home}}.profile"
     line: >
-        alias runtests{{ (10 * item.0)|round|int }}-{{ item.1 }}='PYTHONPATH=/home/vagrant/djangodata/
-        tox -c /django/tox.ini -e py{{ (10 * item.0)|round|int }}{{ ("-" + item.1)|replace("-sqlite3", "") }} --
-        --settings=test_{{ item.1 }}'
+        alias runtests{{ (10 * item.0)|round|int }}-{{ item.1["jenkins_runner"] }}='PYTHONPATH=/home/vagrant/djangodata/
+        tox -c /django/tox.ini -e py{{ (10 * item.0)|round|int }}{{ ("-" + item.1["tox_env_factor"])|replace("-sqlite3", "") }} --
```

Comment:
Just looking quickly but is the replace still required here?

Reply:
I think it's still required, yes. Becvaue the tox test enviroment factor as pero the Django `tox.ini` is still `sqlite3` (untouched by us).

I'm trying to change the Jinja2 constructs in this region. Will push the commit here for your review. If it actually simplifies/makes this more robust then maybe we can address #10 too with this PR.

Reply:
I realise now maybe you were talking about a somewhat more involved change. I've pushed 82007736ee694c8afe0a95f490ebb42fae610f37 which takes advantage of the fact we now have two decoupled DB config Ansible playbook vars.